src/helpers/parse_snak.ts

```diff
@@ -98,7 +98,7 @@ export const parsers = {
   'wikibase-sense': entity,
 } as const
 
-export function parseClaim (datatype, datavalue, options, claimId) {
+export function parseSnak (datatype, datavalue, options) {
```

Comment:
This would need more work due to the hacks on datatype this function does:
```js
  // Known case of missing datatype: form.claims, sense.claims
  datatype = datatype || datavalue.type
  // Known case requiring this: legacy "muscial notation" datatype
  datatype = datatype.replace(' ', '-')
  ```
  I would let that for another PR \^\^

Reply:
Actually, this seems to work a01f5ab, but hacky ><

Reply:
Further addressed in 0805fee